pubtools/pulplib/_impl/model/unit/file.py

```diff
@@ -31,6 +34,46 @@ class FileUnit(Unit):
     )
     """SHA256 checksum of this file, as a hex string."""
 
+    description = pulp_attrib(
+        type=str,
+        pulp_field="pulp_user_metadata.description",
+        default=None,
+        validator=optional_str,
+    )
+    """A user-oriented terse description of this file.
+
+    .. versionadded:: 2.20.0
```

Comment:
I've actually put the wrong version number here when creating the PR, however it might become correct by the time this is submitted, as 2.19.0 seems ready for release.  Maybe ignore this in the review and I'll ensure they're correct at merge time.